### PR TITLE
fix Newsletter Queue method for a case with invalid start_at param

### PR DESCRIPTION
**Pull request by karyna-t**
### Description (*)
Fix for a Newsletter Queue method, which causes a fatal error in integration tests (on PHP 8.0 env)

### Related Issues (if relevant)
1. magento/magento2#34020

### Contribution checklist (*)
 - [ ] Pull request has a meaningful description of its purpose
 - [ ] All commits are accompanied by meaningful commit messages
 - [ ] All new or changed code is covered with unit/integration tests (if applicable)
 - [ ] README.md files for modified modules are updated and included in the pull request if any [README.md predefined sections](https://github.com/magento/devdocs/wiki/Magento-module-README.md) require an update
 - [ ] All automated tests passed successfully (all builds are green)

### CODE REVIEW

**Review by andrewbess — APPROVED** [review]

**Review by eduard13 — APPROVED** [review]

**Comment by andrewbess** [comment]
@magento run all tests

**Comment by andrewbess** [comment]
@magento run all tests with env PHP 8.0

File: app/code/Magento/Newsletter/Model/Queue.php [hunk 1]
```diff
@@ -199,10 +200,18 @@ public function isNew()
      *
      * @param string|null $startAt start date of the mailing queue
      * @return $this
+     * @throws InvalidArgumentException
      */
     public function setQueueStartAtByString($startAt)
     {
-        if ($startAt === null || $startAt == '') {
+        if (!(is_string($startAt) || $startAt === null)) {
```
Comment (xmav) [inline app/code/Magento/Newsletter/Model/Queue.php]: new exceptions are not allowed here. We may try to change logic to call `$this->setQueueStartAt($this->utcConverter->convertLocalizedDateToUtc($startAt))` when `$startAt` is not empty string and set null for all other cases.
If there is issue that comes from` $this->utcConverter->convertLocalizedDateToUtc($startAt)` which may return false we need to handle that two

**Review by xmav — CHANGES_REQUESTED** [review]

**Comment by andrewbess** [comment]
@magento run all tests with env PHP 8.0

**Comment by andrewbess** [comment]
@magento run all tests

File: app/code/Magento/Newsletter/Model/Queue.php (hunk 1 above)
Comment (andrewbess) [reply app/code/Magento/Newsletter/Model/Queue.php]: Hello @xmav 
Thank you for your review
The logic has been fixed, tests too.
Examples for possible conditions are provided [here](https://3v4l.org/cA5GX)

**Comment by andrewbess** [comment]
@magento run Integration Tests with env PHP 8.0

**Comment by andrewbess** [comment]
@magento run Integration Tests

**Review by Den4ik — CHANGES_REQUESTED** [review]
Please look at failed integration tests

**Comment by karyna-tsymbal-atwix** [comment]
@magento run all tests

**Comment by karyna-tsymbal-atwix** [comment]
@magento run all tests with env PHP 8.0

**Comment by andrewbess** [comment]
@magento run Integration Tests, Unit Tests, Static Tests with env PHP 8.0

**Comment by andrewbess** [comment]
@magento run Integration Tests, Unit Tests, Static Tests

**Comment by xmav** [comment]
@magento run Integration Tests, Static Tests with env PHP 8.0

**Pull request merged**